extras/docker/fromsource/heketi-start.sh

```diff
@@ -74,4 +85,32 @@ if [[ -d "${BACKUPDB_PATH}" ]]; then
     fi
 fi
 
-/usr/bin/heketi --config=/etc/heketi/heketi.json
+# if NEED_TOPOLOGY_LOAD is not empty, start the service in the background, and
+# load the topology.json from the path in HEKETI_TOPOLOGY_FILE
+if [ -z "${NEED_TOPOLOGY_LOAD}" ]; then
```

Comment:
We should use bash-style double brackets to be consistent with the rest of the script.

Reply:
Ok, I can change that.

Reply:
NEED_TOPOLOGY_LOAD needs to be in lower case and local to the script.

Reply:
This should have been addressed now.